app/frontend/models/user.ts

```diff
@@ -121,6 +124,8 @@ export const UserModel = types
         newRole = EUserRoles.reviewer
       } else if (self.role === EUserRoles.reviewer) {
         newRole = EUserRoles.reviewManager
+      } else if (self.role === EUserRoles.technicalSupport) {
```

Comment:
Are we sure we want to be able to change a tech support to a review/manager, but not back?